### ISSUE TITLE
Constants XXX should be exposed as static fields instead of getXXX()

### ISSUE THREAD

**Issue by Fiouz** [asciidoctor/asciidoctor-gradle-plugin#367]
# Summary

JVM constants should be directly exposed as `public` fields instead of exposing intrinsic `public static getXXX()` getter.

# Details

At https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/e5393078d15f6e90acab27ac8c05be6d5773a04a/asciidoctor-gradle-jvm/src/main/groovy/org/asciidoctor/gradle/jvm/AsciidoctorJExtension.groovy#L64

due to the lack of `public` modifier, the Groovy compiler generates a `public static String getDEFAULT_DIAGRAM_VERSION()` getter backed by a `private static final String DEFAULT_DIAGRAM_VERSION` class variable (this is mentioned at [https://mrhaki.blogspot.com/2015/09/groovy-goodness-defining-public.html](https://mrhaki.blogspot.com/2015/09/groovy-goodness-defining-public.html)).

Relevant [ASM](https://asm.ow2.io/) output:
```
  // access flags 0x9
  public static getDEFAULT_DIAGRAM_VERSION()Ljava/lang/String;
  @Lgroovy/transform/Generated;()
    GETSTATIC org/asciidoctor/gradle/jvm/AsciidoctorJExtension.DEFAULT_DIAGRAM_VERSION : Ljava/lang/String;
    ARETURN
    MAXSTACK = 1
    MAXLOCALS = 0
```
```
  // access flags 0x1A
  private final static Ljava/lang/String; DEFAULT_DIAGRAM_VERSION = "1.5.16"
```

This leads to cumbersome indirection/notation (that is hidden to Groovy scripts):
```kotlin
// build.gradle.kts

pluginManager.withPlugin("org.asciidoctor.jvm.base") {
    extensions.configure(AsciidoctorJExtension::class.java) {
        setDiagramVersion(AsciidoctorJExtension.getDEFAULT_DIAGRAM_VERSION())
    }
}
```

As exposing the static fields through getters does not seem the original intent, could it be simplified to a simple `public` static field for 3.x? (Since removing the getters would introduce a binary incompatible change for the current 2.x branch).

**Comment by ysb33r** [comment]
@aalmiray  I'll take care of this.

@Fiouz Yes, that is a definite oversight. 

And BTW As from 2.2.0 you can activate diagram simply by doing `modules.getDiagram().use()` and it will activate the default diagram version.

**Comment by ysb33r** [comment]
Fixed in 3.0.0-alpha.1
